R/fotmob_leagues.R

```diff
@@ -142,16 +142,6 @@ fotmob_get_league_ids <- function(cached = TRUE) {
   )
 }
 
-#' @importFrom stringr str_replace
-.fotmob_get_league_resp_from_build_id <- function(page_url, stats = FALSE) {
```

Comment:
outdated function only used in code I'm removing from the `fotmob_get_season_stats()` internals